### ISSUE TITLE
Markup in page titles ends up with HTML in the <title> tag

### ISSUE THREAD

**Issue by hynek** [pradyunsg/furo#26]
This page https://github.com/python-attrs/attrs/blob/master/docs/examples.rst has a title of

```rst
``attrs`` by Example
====================
```

Which results in the following html title tag:

```html
<title><code class="docutils literal notranslate"><span class="pre">attrs</span></code> by Example - attrs 20.3.0.dev0 documentation</title>
```

(you can check for yourself on https://www.attrs.org/en/latest/examples.html :))

**Comment by pradyunsg** [comment]
OH interesting!

/me goes to look at what other sphinx themes do.

**Comment by pradyunsg** [comment]
Fixed in 861e7fd.
